packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -41,13 +41,25 @@ $Rating-largeIconSize: 20px;
 
     color: $Rating-defaultColor;
 
+    @media screen and (-ms-high-contrast: active) {
+      color: WindowText;
+    }
+
     &.starIsSelected {
       color: $Rating-selectedColor;
+
+      @media screen and (-ms-high-contrast: active) {
+        color: Highlight;
```

Comment:
@Jahnp @betrue-final-final, the issue with that is that in about half of the default HC palettes `Window` and `HighlightText` are the same color, so `color: HighlightText;` should never be used unless `background: Highlight;` is also set.

There may be a few other places where I've set text color to `Highlight` to indicate some sort of selection or emphasis, and I'll double check if any of those are inappropriate. However, this particular instance in Rating setting color changes the color of the filled in star characters, which is arguably more like a fill color than a text color, so I feel `Highlight` is the correct color in this instance.